client/jetpack-cloud/sections/agency-signup/agency-signup-form/index.tsx

```diff
@@ -78,17 +78,17 @@ export default function AgencySignupForm() {
 	// Redirect the user to the dashboard if they are already a partner,
 	// or the overview page if the form was submitted successfully,
 	// or the issue licenses page if coming via the /manage/pricing page.
+	const source = useRef( queryParams.get( 'source' ) );
+	const bundleSize = useRef( queryParams.get( 'bundle_size' ) );
+	const products = useRef( queryParams.get( 'products' ) );
 	useEffect( () => {
 		if ( createPartner.isSuccess ) {
-			if ( queryParams.get( 'source' ) === 'manage-pricing-page' ) {
-				const bundleSize = queryParams.get( 'bundle_size' ) || '1';
-				const path = `/partner-portal/issue-license?product_slug=${ queryParams.get(
-					'products'
-				) }&bundle_size=${ bundleSize }&source=manage-pricing-page`;
+			if ( source.current === 'manage-pricing-page' ) {
```

Comment:
Not a blocker, but this would be a bit more robust:

```suggestion
			if ( source.current === 'manage-pricing-page' && bundleSize.current && products.current ) {
```

Reply:
True, but then perhaps it is better not to prevent the redirect prematurely, if the bundleSize and products params aren't there for any reason? At least it would still get to the issue licenses page then :D

Reply:
Fair point, and I like that! My concern was having `null` in the URL (tested on my end), but in the end that's not a big deal.